### ISSUE TITLE
Doc:  Integrate the existing Kibana with Devtron dashboard

### ISSUE THREAD

**Issue by Abhinav-26** [devtron-labs/devtron#2483]
### 💭 Description

Add a new use case for integrating the existing Kibana with the Devtron dashboard under the [Use Cases](https://docs.devtron.ai/resources/use-cases) section of Devtron documentation.

If you have any queries, please do not hesitate to ask in our [discord community](https://discord.devtron.ai/) in `#hacktoberfest` channel.

Feel free to add more use-cases.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/devtron-labs/devtron/blob/main/CODE_OF_CONDUCT.md)

AB#678

**Comment by Suraj-kumar00** [comment]
Hi @Abhinav-26, is this issue open as it seems to me? Like can I work on this.

**Comment by Abhinav-26** [comment]
> Hi @Abhinav-26, is this issue open as it seems to me? Like can I work on this.

Hey @Suraj-kumar00 yes, it's still open. You can work on this use-case and rasie a PR for docs.

**Comment by Suraj-kumar00** [comment]
Yeah thanks @Abhinav-26, can you please add hacktoberfest label here?

**Comment by Suraj-kumar00** [comment]
@Abhinav-26 Can you please elaborate more about it?

**Comment by Atharva1723** [comment]
@Abhinav-26  Can I work on this? Please can you add me as assignee

**Comment by Abhinav-26** [comment]
> @Abhinav-26 Can I work on this? Please can you add me as assignee

Hey, you can work on other use-cases. @Suraj-kumar00 is already working on it.
Please [refer to this](https://devtron.ai/blog/dive-into-open-source-joy-with-devtron-in-hacktoberfest-2023/) for more details how you can contribute to Devtron

**Comment by Suraj-kumar00** [comment]
> Yeah thanks @Abhinav-26, can you please add hacktoberfest label here?

Hi @Abhinav-26,  can you please add hacktoberfest label to this issue?

**Comment by Abhinav-26** [comment]
> > Yeah thanks @Abhinav-26, can you please add hacktoberfest label here?
> 
> Hi @Abhinav-26, can you please add hacktoberfest label to this issue?

hey, it's done. Btw repo needs the hactoberfest label. Just FYI

**Comment by zeborg** [comment]
Hey @Abhinav-26, since this issue has remained open for almost a year please let me know if it's up for grabs yet.

**Comment by visionaryfire** [comment]
Hi @Abhinav-26 Is this issue still on its lifecycle?

**Comment by Abhinav-26** [comment]
Hello @zeborg, yes. You can pick this up and work on it. 

P.S: Devtron does participate in hacktoberfest but doesn't roll out swags for contributions.
